<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XSessionConfigurationWidgetBase</class>
 <widget class="QWidget" name="XSessionConfigurationWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>X Session Log File</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&amp;Log file:</string>
          </property>
          <property name="buddy">
           <cstring>xsessionUrlRequester</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KUrlRequester" name="xsessionUrlRequester">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="ignoreXorgErrors">
        <property name="toolTip">
         <string>Check this option to remove Xorg errors</string>
        </property>
        <property name="whatsThis">
         <string>Check this option to remove Xorg errors</string>
        </property>
        <property name="text">
         <string>Ignore Xorg errors</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="xorgErrorsDescription">
        <property name="text">
         <string>&lt;p&gt;&lt;b&gt;Note:&lt;/b&gt; Check this option to remove X.org errors.&lt;/p&gt;&lt;p style=&quot;margin-bottom:0px&quot;&gt;The following lines will be ignored:&lt;/p&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
